fhir-persistence/src/main/java/com/ibm/fhir/persistence/FHIRPersistence.java

```diff
@@ -148,7 +148,7 @@
      *         an OperationOutcome with hints, warnings, or errors related to the interaction
      * @throws FHIRPersistenceException
      */
-    <T extends Resource> MultiResourceResult<T> history(FHIRPersistenceContext context, Class<T> resourceType, String logicalId) throws FHIRPersistenceException;
+    MultiResourceResult history(FHIRPersistenceContext context, Class<? extends Resource> resourceType, String logicalId) throws FHIRPersistenceException;
```

Comment:
I think I had used a generic here to help avoid an unchecked cast while converting the results to the expected resource type (since the history resources will all be of the same type).  However, I agree that would get more complicated now that the ResourceResult contains a resource wrapper and not the resource themselves.
Net:  I think I'm fine with this change.

Reply:
Thought about this for a while and in the end I felt it didn't make sense to make it more generic than it is...because it's not like we can return anything other than a collection of Resources, and those resources may also be different types (of Resource)